dali/test/python/test_copy_to_external_torch.py

```diff
@@ -12,36 +12,39 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import torch
-import torch.utils.dlpack as torch_dlpack
 import ctypes
 import numpy as np
-import nvidia.dali as dali
 import nvidia.dali.fn as fn
+import torch
 from nvidia.dali import pipeline_def
-from nvidia.dali.backend import TensorGPU, TensorListGPU
 from nvidia.dali import types
+from nvidia.dali.backend import TensorListGPU
 
 shape = [4000000]
 batch_size = 2
 
+
 @pipeline_def(batch_size=batch_size, device_id=0, num_threads=8)
 def _test_pipe():
-    get_tensor = lambda si: np.arange(si.idx_in_epoch, si.idx_in_epoch + shape[0], dtype=np.int32)
+    def get_tensor(si):
+        return np.arange(si.idx_in_epoch, si.idx_in_epoch + shape[0], dtype=np.int32)
+
     inp = fn.external_source(get_tensor, batch=False)
     return inp.gpu()
 
+
 to_torch_type = {
-    types.DALIDataType.FLOAT   : torch.float32,
-    types.DALIDataType.FLOAT64 : torch.float64,
-    types.DALIDataType.FLOAT16 : torch.float16,
-    types.DALIDataType.UINT8   : torch.uint8,
-    types.DALIDataType.INT8    : torch.int8,
-    types.DALIDataType.INT16   : torch.int16,
-    types.DALIDataType.INT32   : torch.int32,
-    types.DALIDataType.INT64   : torch.int64
+    types.DALIDataType.FLOAT: torch.float32,
+    types.DALIDataType.FLOAT64: torch.float64,
+    types.DALIDataType.FLOAT16: torch.float16,
+    types.DALIDataType.UINT8: torch.uint8,
+    types.DALIDataType.INT8: torch.int8,
+    types.DALIDataType.INT16: torch.int16,
+    types.DALIDataType.INT32: torch.int32,
+    types.DALIDataType.INT64: torch.int64
```

Comment:
Is that change really necessary?

Reply:
Yes, I believe the linter has complained here